project/plugins.sbt

```diff
@@ -3,25 +3,15 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefact
   Resolver.ivyStylePatterns
 )
 
-sys.env.get("PLAY_VERSION") match {
-  case Some("2.8") => // required since we're cross building for Play 2.8 which isn't compatible with sbt 1.9
-    libraryDependencySchemes += "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always
-  case _           => libraryDependencySchemes := libraryDependencySchemes.value // or any empty DslEntry
-}
-
-sys.env.get("PLAY_VERSION") match {
-  case Some("2.8") => addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.20")
-  case Some("2.9") => addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.9.0")
-  case _           => addSbtPlugin("org.playframework" % "sbt-plugin" % "3.0.0")
-}
+addSbtPlugin("org.playframework" % "sbt-plugin" % "3.0.0")
```

Comment:
no point cross building a library that changes so little, if a new play version comes along, just update it and if you need to patch old versions while people are still updating then you can do that